Patches/Racc/Race_Racc.xml

```diff
@@ -94,33 +97,11 @@
 			</tools>
 			</value>
 			</li>
-
+		</operations>
+	</match>
 	<!-- ================================================= -->
 	<!-- === BaseBodySize, ArmorRatings, MoveSpeed, and BaseHealth should already be set. I think. === -->
 	<!-- ===================================== -->
-	<!-- === Adds In Inspector Tabs 	   === -->
-	<!-- === Inventory and all that jazz   === -->
-	<!-- === Again, no need to change 'em  === -->
-	<!-- === Unless you changed it         === -->
-	<!-- === then change 'em			   === -->
-	
-	<li Class="PatchOperationReplace">
-		<xpath>Defs/ThingDef[@Name="BaseRaccPawn"]/inspectorTabs/li[.="ITab_Pawn_Gear"]</xpath>
-		<value>
-			<li>CombatExtended.ITab_Inventory</li>
-		</value>
-	</li>
-
-	<li Class="PatchOperationAdd">
-	
-		<xpath>Defs/ThingDef[@Name="BaseRaccPawn"]/comps</xpath>
-		<value>
-			<li Class="CombatExtended.CompProperties_Inventory" />
-		</value>
-	</li>
-		</operations>
-		</match>
-	</Operation>

```

Comment:
Very close, but the patch still crashes. I checked, and it just needs this `</Operation>` back.

Reply:
Should be fixed now, was tired when I wrote it :|